cpp/ql/test/library-tests/templates/isfromtemplateinstantiation/isfromuninstantiatedtemplate.expected

```diff
@@ -100,20 +96,37 @@ isFromUninstantiatedTemplate
 | isfromtemplateinstantiation.cpp:93:55:93:55 | return ... | isfromtemplateinstantiation.cpp:93:7:93:15 | myMethod1 |
 | isfromtemplateinstantiation.cpp:94:7:94:15 | declaration of myMethod2 | isfromtemplateinstantiation.cpp:77:26:77:45 | AnotherTemplateClass<T> |
 | isfromtemplateinstantiation.cpp:94:7:94:15 | declaration of myMethod2 | isfromtemplateinstantiation.cpp:97:25:97:60 | myMethod2 |
+| isfromtemplateinstantiation.cpp:94:7:94:15 | declaration of myMethod2 | isfromtemplateinstantiation.cpp:97:52:97:52 | myMethod2 |
```

Comment:
I'd like @jbj to look at these changes at some point (no need to hold back merging though).

Reply:
@ian-semmle and @nickrolfe, have you checked that the test changes in `isfromtemplateinstantiation/*.expected` are what they should be? These tests are very sensitive to exactly the things you changed in the discover branch, and they contain some of the edge cases that have been problematic in the past.

Reply:
The changes look OK to me. There's a new `| I | T |` row in isfromuninstantiatedtemplate, but that's probably the same issue as the existing rows.